packages/preview/simple-ecnu-thesis/0.1.0/package/font.typ

```diff
@@ -0,0 +1,75 @@
+// --------------------- 字号 ---------------------
+#let 初号 = 42pt
+#let 小初 = 36pt
+#let 一号 = 26pt
+#let 小一 = 24pt
+#let 二号 = 22pt
+#let 小二 = 18pt
+#let 三号 = 16pt
+#let 小三 = 15pt
+#let 四号 = 14pt
+#let 中四 = 13pt
+#let 小四 = 12pt
+#let 五号 = 10.5pt
+#let 小五 = 9pt
+#let 六号 = 7.5pt
+#let 小六 = 6.5pt
+#let 七号 = 5.5pt
+#let 小七 = 5pt
+
+// --------------------- 具体的字体 ---------------------
+// 英文字体
+#let Times-New-Roman = "Times New Roman"
```

Comment:
The most common convention for variable names in Typst package is lowercase-kebab-case. It would be nice if you could rename the few variables in this file so that they don't have capital letters.